go/vt/vtgate/plan_execute.go

```diff
@@ -64,11 +64,11 @@ func (e *Executor) newExecute(
 	logStats *logstats.LogStats,
 	execPlan planExec, // used when there is a plan to execute
 	recResult txResult, // used when it's something simple like begin/commit/rollback/savepoint
-) error {
-	// 1: Prepare before planning and execution
+) (err error) {
```

Comment:
Any reason why we have error as a named parameter here? That is not our usual normal practice.

Reply:
To be sure that we're checking the returned error in the defer.